upload-action/src/filecoin.js

```diff
@@ -284,35 +284,48 @@ export async function uploadCarToFilecoin(synapse, carPath, ipfsRootCid, options
   const uploadResult = await executeUpload(synapseService, carBytes, cid, {
     logger,
     contextId: `gha-upload-${Date.now()}`,
-    ipniValidation: {
-      enabled: true,
-      onProgress: (event) => {
-        if (event.type === 'ipniAdvertisement.retryUpdate') {
+    onProgress: (event) => {
```

Comment:
@rvagg you can see the difference in onProgress and callbacks here.. super easy to pipe through.. Kudos to Alex for showing it first in Helia. typesafe events and data types as well. It's pretty nice.